.github/workflows/CI.yml

```diff
@@ -18,10 +18,21 @@ jobs:
           - 'nightly'
         os:
           - ubuntu-latest
-          - macOS-latest
+          - macos-13 # Intel
           - windows-latest
         arch:
           - x64
```

Comment:
IMO the easiest is to just remove `arch` and always use the runner's architecture (the default if not specified):
```suggestion
```

Reply:
The `arch` setting of `setup-julia` has to be removed as well below.